dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -531,19 +536,41 @@ const backgroundSpeechBubble = (format: ArticleFormat): string => {
 };
 
 const fillCommentCount = (format: ArticleFormat): string => {
+	if (
+		format.design === ArticleDesign.LiveBlog ||
+		format.design === ArticleDesign.DeadBlog
+	)
+		return neutral[46];
 	if (format.theme === ArticleSpecial.Labs) return BLACK;
 	if (format.theme === ArticleSpecial.SpecialReport)
 		return specialReport[300];
 	return pillarPalette[format.theme].main;
 };
 
+const fillCommentCountUntilDesktop = (format: ArticleFormat): string => {
+	if (format.design === ArticleDesign.LiveBlog) return WHITE;
+
+	return fillCommentCount(format);
+};
+
 const fillShareIcon = (format: ArticleFormat): string => {
 	if (format.theme === ArticleSpecial.Labs) return BLACK;
 	if (format.theme === ArticleSpecial.SpecialReport)
 		return specialReport[300];
 	return pillarPalette[format.theme].main;
 };
 
+const fillShareCountIcon = neutral[46];
```

Comment:
```suggestion
const fillShareCountIcon = (): string => {
	return neutral[46];
};
```

What do you think about maintaining the function pattern? I think this will make the code easier to reason about and scale if everything is consistent and it should make moving to importing direct functions easier as well. If I'm importing functions _and_ variables then it will be hard to know when I need to execute them or not.

Reply:
Great idea Oliver, thanks. I also think it'd be easier for the consumer if everything is function so they know it always needs to be executed 👍